### DISCUSSION TITLE
Querying for null values

### DISCUSSION THREAD

**Discussion by ericchernuka** [mswjs/data#186] [Q&A] [answered]
I have a model that looks like this:
```typescript
definition: {
  id: primaryKey(faker.datatype.uuid),
  organizationId: nullable((): string | null => null)
}
```

I then want to query for any definition that has a null organizationId, but it seems that the query will only let me look for strings or undefined.
![image](https://user-images.githubusercontent.com/243155/151097214-492631b6-a049-4597-bb18-39624cd5bb24.png)

Is there a way to query for null values?

**Comment by kettanaito** [comment, marked as answer]
Hi, @ericchernuka. Thank you for raising this.

This looks like an issue on our side. I've opened a pull request to tackle it (https://github.com/mswjs/data/pull/187). Currently added the failing test similar to what you're doing in the example above. Should be fixed by aligning the query types to support `nullable` properties.
